formiodata/components/__init__.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Nova Code (http://www.novacode.nl)
+# See LICENSE file for full licensing details.
+
+# basic
+from .textfield import textfieldComponent
+from .textarea import textareaComponent
+from .number import numberComponent
+from .password import passwordComponent
+from .checkbox import checkboxComponent
+from .selectboxes import selectboxesComponent
+from .select import selectComponent
+from .radio import radioComponent
+from .button import buttonComponent
+# advanced
+from .email import emailComponent
+from .url import urlComponent
+from .phoneNumber import phoneNumberComponent
+from .address import addressComponent
+from .datetime import datetimeComponent
+from .day import dayComponent
+from .time import timeComponent
+from .currency import currencyComponent
+from .survey import surveyComponent
+from .signature import signatureComponent
+from .htmlelement import htmlelementComponent
+from .content import contentComponent
+# layout
+from .columns import columnsComponent
+from .fieldset import fieldsetComponent
+from .panel import panelComponent
+from .table import tableComponent
+from .tabs import tabsComponent
+# data and grid
+from .datagrid import datagridComponent
+from .editgrid import editgridComponent
+# other, premium
+from .file import fileComponent
+from .resource import resourceComponent
```

Comment:
Are these imports needed? If I understand the code correctly, it tries to import `<componentTypeClass>` from `components.<type>`. These imports make `components.<componentTypeClass>` available, which is not necessary (except maybe for backwards compatibility for when users are importing the classes directly).

Reply:
@sjamaan I updated my previous comment with a better traceback.

Reply:
Such imports like ...

```
File "/home/bob/src/python-formio-data/tests/test_nested_components.py", line 12, in <module>
  from formiodata.components import (
```

Reply:
You'd have to change the imports in the tests from

```python
from formiodata.components import (
   columnsComponent,
   datetimeComponent, 
   ...
)
```
to

```python
from formiodata.components.columns import columnsComponent
from formiodata.components.datetime import datetimeComponent
...
```